client/src/views/FightView.tsx

```diff
@@ -89,8 +91,26 @@ const FightView = () => {
             ))}
           </Box>
           {/* FIGHT */}
-          <Box sx={{ ml: 5, alignSelf: 'center' }}>
+          <Box sx={{ ml: 5, alignSelf: 'center', position: 'relative' }}>
+            {/* Fight Component */}
             <FightComponent fight={fight} />
+            {currentMapEffect === '10.png' && (
```

Comment:
Instead of checking for a single background, you have to find the current background object from the fightBackgrounds list, and find its effect.
If there is none, display nothing, if there is one, select the correct thing to display

Reply:
I looked into this, and only solution I see is to alter model Fight in database to also hold .effect. Or maybe it is possible to import out of scope of /core but I don't see a way how to do it